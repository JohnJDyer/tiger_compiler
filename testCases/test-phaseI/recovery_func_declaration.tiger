let
  function a(a: int, b: float);
  function b(): ;
  function c(): int begin;
  function d(): int begin end;
in
  flush();
end