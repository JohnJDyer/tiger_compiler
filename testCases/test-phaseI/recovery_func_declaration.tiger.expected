a:
b:
c:
d:
main:
	call, flush
	return, , , 
